src/query/dump.cpp

```diff
@@ -536,6 +552,25 @@ PullPlanDump::PullChunk PullPlanDump::CreateInternalIndexCleanupPullChunk() {
   };
 }
 
-void DumpDatabaseToCypherQueries(query::DbAccessor *dba, AnyStream *stream) { PullPlanDump(dba).Pull(stream, {}); }
+PullPlanDump::PullChunk PullPlanDump::CreateTriggersPullChunk() {
+  return [this](AnyStream *stream, std::optional<int>) {
+    if (trigger_store_->HasTriggers()) {
```

Comment:
No need for check, if there are none `GetTriggerInfo` will return empty vector and you will iterate over empty vector